GliaWidgets/Sources/Screensharing/ScreenShareHandler.Implementation.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+extension ScreenShareHandler {
+    static func create() -> ScreenShareHandler {
```

Comment:
I think normally these are being called `live` rather than `create`. Especially as the mock is called `mock`. Also is there a way this is a variable rather than a function? Like the mock https://github.com/salemove/ios-sdk-widgets/pull/562/files#diff-a06be315abeb7eff822e44608e9f208a6d718246da45d7b116beb7a563621d2dR5.

Reply:
We had discussion with @igorkravchenko about that. 
@igorkravchenko could you share our agreement about that?